File: easy/lua/onboarding.lua
while true do
    local enemy1 = io.read()
    local dist1 = tonumber(io.read())
    local enemy2 = io.read()
    local dist2 = tonumber(io.read())

    if dist1 < dist2 then
        print(enemy1)
    else
        print(enemy2)
    end
end

File: easy/clojure/onboarding.clj
(defn -main []
  (while true
    (let [enemy1 (read-line)
          dist1 (Integer/parseInt (read-line))
          enemy2 (read-line)
          dist2 (Integer/parseInt (read-line))]
      (if (< dist1 dist2)
        (println enemy1)
        (println enemy2)))))
